src/System.IO.Packaging/src/System/IO/Packaging/ZipPackage.cs

```diff
@@ -538,7 +538,7 @@ static private CompressionOption GetCompressionOptionFromZipFileInfo(ZipArchiveE
         private FileAccess _packageFileAccess;
         private FileMode _packageFileMode;
 
-        private static readonly string s_forwardSlash = "/"; //Required for creating a part name from a zip item name
+        private const string ForwardSlash = "/"; //Required for creating a part name from a zip item name
```

Comment:
Should this be named ForwardSlashString and defined in the same place as ForwardSlashChar and s_forwardSlashCharArray?